Runtime/Scripts/CarrierInfoIntegration.cs

```diff
@@ -170,19 +193,19 @@ public AndroidJavaObject GetTelephonyManager()
       return telManager;
     }
 
-    public int GetSignalStrength()
+    public ulong GetSignalStrength()
```

Comment:
This is primarily an Android thing, so it should match what the Android SDK is returning. (0-4, which -1 being not able to return anything, unless 5 has a meaning. There are no unsigned integers in Java.

Reply:
Related API doc: https://developer.android.com/reference/android/telephony/SignalStrength#getLevel()

Reply:
Created a bug to track that https://mobiledgex.atlassian.net/browse/EDGECLOUD-5743